Core/Src/dti.c

```diff
@@ -90,12 +117,14 @@ void dti_set_current(int16_t current)
 
 void dti_set_brake_current(int16_t brake_current)
 {
-	can_msg_t msg = { .id = 0x056, .len = 2, .data = { 0 } };
-
+	can_msg_t msg = { .id = 0x056,
+			  .len = 8,
+			  .data = { 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0 } };
```

Comment:
I will fix calypso so this isnt needed once I get back.

Reply:
Ok I'll keep it for now and fix it later just let me know when I should